example/django.nV/taskManager/loop_false_negative.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from tempfile import NamedTemporaryFile
+
+from django.shortcuts import redirect
+from django.http import HttpResponse
+
+def download(request):
+    response = HttpResponse("Hi.")
+    fork_list = request.POST.getlist('fork_list')
+    if request.POST and len(fork_list) > 0:
+        tmp_file = NamedTemporaryFile()
+        cmd = "tar -czvf %s -C %s " % (tmp_file.name,DOWNLOADS)
+        for item in fork_list:
```

Comment:
So fork_list is tainted but item isn't, problematic. Solved by turning it into an assignment so reaching definitions can notice it.